vMenu/menus/VehicleOptions.cs

```diff
@@ -1796,7 +1796,7 @@ public void UpdateMods(int selectedIndex = 0)
                     "Street"        // 11
                 };
                 MenuListItem vehicleWheelType = new MenuListItem("Wheel Type", wheelTypes, MathUtil.Clamp(GetVehicleWheelType(veh.Handle), 0, 11), $"Choose a ~y~wheel type~s~ for your vehicle.");
-                if (!veh.Model.IsBoat && !veh.Model.IsHelicopter && !veh.Model.IsPlane && !veh.Model.IsBicycle && !veh.Model.IsTrain)
+                if (!veh.Model.IsBoat && !veh.Model.IsBicycle && !veh.Model.IsTrain)
```

Comment:
I'm going to have to change this to not rely on indexes, but rather just provide the vehicle mod id as a menu item data value.

Reply:
I understand. I've debated this before on another project, but ended up keeping the indexes. Pure laziness. ;-) 

Line 1799 was added in v3.2.1 was there a reason for the restriction?

Reply:
Because helicopters and planes don't have different wheel types, and it caused problems with the new open wheel class.